data/backend/authentication/forms.py

```diff
@@ -0,0 +1,76 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.forms import ModelForm
+from django.utils.translation import gettext_lazy as _
+from mailauth.backends import MailAuthBackend
+from mailauth.forms import EmailLoginForm as MailauthEmailLoginForm
+import json
+import os
+import sys
```

Comment:
Nit: usually std lib imports should be at the start of the import list

We may want to rely on a python code formatter like [Ruff](https://docs.astral.sh/ruff/) which also auto-sort imports, so we avoid annoying code style comments like this one (think of Ruff as elm-format for Python).

Reply:
Just discovered ruff which is a must-have. I need to improve my nvim setup to use ruff, and auto sort the imports.